Basic-Scripts/Simple_calculator.py

```diff
@@ -26,4 +26,132 @@
     print(num / num2)
 
 else:
-    print('Wrong Input! Please Try Again')
+    print('Wrong Input! Please Try Again")
+
+
+
+
+
+"""The previous script becomes very boring for the user as he has to keep on typing just numbers and of wants to interact using 
+simple english language he is unable to do so. Also the program terminates after one execution and if the user wants to keep on 
+performing operations it becomes difficult.
+I have tried making a more simple and interactive calculator which allows the user to input any statement for performing the tasks. """
+
+
+
+
+def Calculator(num_1,num_2):
+    #Executing the loop infinite times as we donot know how many times the user will want to run
+    while(True):
+        choice= input("Enter what you want to perform: ")

```

Comment:
Ask user for input by providing the allowed actions (addition, subtraction etc.)